docs/developers/guides/coin_token_registration.md

```diff
@@ -52,9 +61,9 @@ The following topics must be addressed when drafting an ERC-20 Proposal:
 
 Remember to provide links to the relevant [Commonwealth Evmos community](https://commonwealth.im/evmos) discussions concerning your proposal, as well as the [proposal on testnet](#submit-the-proposal-to-the-testnet).
 
-## Submitting the ERC-20 Proposal
+## Submitting the Registration Proposal
```

Comment:
The cli command in this section is wrong its missing `register-coin` and `register-erc20`, e.g. ` evmosd tx gov submit-proposal register-erc20`